xml/ftp_banners.xml

```diff
@@ -241,19 +243,6 @@ example.com FTP server (Version:  Mac OS X Server) ready.</example>
     <param pos="1" name="host.name"/>
   </fingerprint>
 
-  <fingerprint pattern="^ProFTPD (\d+\.[^\s]+) Server \(Linksys(.*)\) \[(.+)\]$">
```

Comment:
This looks like a duplicate of what's above. I can't find any examples that don't start with W (WRTxxx). Also, it fails if I keep it since the example in this one was matching the earlier fingerprint and causing `bundle exec rake tests` to fail.

Reply:
The regex in both of the fingerprints remains unchanged since the initial commit 8 years ago. As you mentioned the only real difference is the `W` in the regex.

I've checked some of our datasets and I do NOT see any matches for this fingerprint in them.
I DO see matches for the `W` variant.

++ to seeing this FP removed.